### PR TITLE
Manual verification for lp-1889555

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

File: bugs/lp-1889555.txt [hunk 1]
```diff
@@ -0,0 +1,1210 @@
...
+
+ubuntu@ip-172-31-32-175:~$ cloud-init --version
+/usr/bin/cloud-init 19.2-24-ge7881d5c-0ubuntu1~18.04.1
+ubuntu@ip-172-31-32-175:~$ lsblk | grep nvme
+nvme0n1     259:0    0  100G  0 disk
+└─nvme0n1p1 259:1    0  100G  0 part /
+
+ubuntu@ip-172-31-32-175:~$ debconf-show grub-pc | grep "install_devices:"
+debconf: DbDriver "passwords" warning: could not open /var/cache/debconf/passwords.dat: Permission denied
+* grub-pc/install_devices: /dev/sda
```
Comment (blackboxsw) [inline bugs/lp-1889555.txt:256]: Is this significant that this bionic image doesn't have a grub-efi debconfig setting?

**Review by blackboxsw — APPROVED** [review]
I think this looks good. Just wondering why grub-efi pkg. You'll likely need to attach this verifcation output  to the bug https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1889555 so that we can clear SRU now.

File: bugs/lp-1889555.txt (hunk 1 above)
Comment (TheRealFalcon) [reply bugs/lp-1889555.txt:256]: I noticed that, but no, the issue only manifests on the grub-pc installs.

File: bugs/lp-1889555.txt [hunk 2]
```diff
@@ -0,0 +1,1210 @@
...
+# Verify no "Reconfiguring grub" message in upgrade output
+# Verify install_devices not changed
+debconf-show grub-pc | grep "install_devices:"
+
+Steps for test 3:
+# lxd launch affected image
+lxc launch <image>
+
+# Obtain bash shell
+lxc exec <image> bash
```
Comment (OddBloke) [inline bugs/lp-1889555.txt:70]:
```suggestion
lxc exec <instance> bash
```

And for future reference (though as you've already done validation with `exec` we shouldn't change it here), you can get a shell with:

```suggestion
lxc shell <instance>
```

**Comment by OddBloke** [comment]
One minor typo found, but otherwise this looks good. Thanks!

**Pull request merged**